src/CampaignsAdmin/CampaignAdmin.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Divider } from 'antd';
+import { CampaignList } from './CampaignList';
+import { NewCampaignForm } from './NewCampaignForm';
+
+
+function CampaignAdmin() {
+  
+    return (
+        <>
+            <Divider style={{ borderTopColor: 'gray' }} orientation="left">
```

Comment:
This shouldn't be using `style`, but use the palette that comes with antd, OR use CSS styles based off of classes instead